git/github/github_test.go

```diff
@@ -195,4 +232,65 @@ var _ = Describe("Client", func() {
 		Expect(err).To(BeNil())
 		Expect(checkRunID).To(BeNil())
 	})
+
+	It("can check if check run updated is needed", func() {
+		var checkRunAdapter = &github.CheckRunAdapter{
+			Name:           "example-name",
+			Owner:          "example-owner",
+			Repository:     "example-repo",
+			SHA:            "abcdef1",
+			ExternalID:     "example-external-id",
+			Conclusion:     "success",
```

Comment:
Conclusion differs with the previous one.
Previously used: **Passed**
Should we set this one also to Passed?